tests/manager/manager-ctor-read_preference-001.phpt

```diff
@@ -0,0 +1,96 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): read preference options
+--FILE--
+<?php
+
+$tests = [
+    ['mongodb://127.0.0.1/?readPreference=primary', []],
+    ['mongodb://127.0.0.1/?readPreference=secondary&readPreferenceTags=tag:one&readPreferenceTags=', []],
+    ['mongodb://127.0.0.1/?readPreference=secondary&maxStalenessMS=1000', []],
+    [null, ['readPreference' => 'primary']],
+    [null, ['readPreference' => 'secondary', 'readPreferenceTags' => [['tag' => 'one'], []]]],
+    [null, ['readPreference' => 'secondary', 'maxStalenessMS' => 1000]],
+];
+
+foreach ($tests as $test) {
+    list($uri, $options) = $test;
+
+    $manager = new MongoDB\Driver\Manager($uri, $options);
+    var_dump($manager->getReadPreference());
+}
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+object(MongoDB\Driver\ReadPreference)#%d (%d) {
+  ["mode"]=>
+  int(1)
```

Comment:
This is going to fail/create conflicts after the PHPC-498 changes (ReadPreference, ReadConcern, and WriteConcern value).